observations/README.md

```diff
@@ -0,0 +1,1279 @@
+# Observation Mapping
+
+Main section links:
+- **[XML HL7 > JSON FHIR](#XML-HL7->-JSON-FHIR)**
+- **[JSON FHIR > XML HL7](#JSON-FHIR->-XML-HL7)**
+- **[Further documentation](#Further-documentation)**
+
+# XML HL7 > JSON FHIR
+
+FHIR Observations are used throughout the Structured Record and the 
+[GP Connect Documentation](https://developer.nhs.uk/apis/gpconnect-1-6-0/accessrecord_structured.html) outlines how they
```

Comment:
We tried clicking this link, but it didn't appear to mention "Observation".  Is that expected?

Reply:
Agreed, I've removed this link completely as the relevant parts of the docs are linked elsewhere